test/e2e/framework/node/wait_test.go

```diff
@@ -31,7 +31,7 @@ import (
 // since single node checks are in TestReadyForTests.
 func TestCheckReadyForTests(t *testing.T) {
 	// This is a duplicate definition of the constant in pkg/controller/service/controller.go
-	labelNodeRoleMaster := "node-role.kubernetes.io/master"
+	labelNodeRoleControlPlane := "node-role.kubernetes.io/controlPlane"
```

Comment:
According to https://github.com/kubernetes/enhancements/blob/165d718f2156873b820cfa1701ca0b0c488c8adf/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint/README.md#renaming-the-node-rolekubernetesiomaster-node-taint I guess there should be `"node-role.kubernetes.io/control-plane"`